graphql-schemas/schema.yml

```diff
@@ -1356,6 +1356,7 @@ confs:
   - { name: deletionApprovals, type: DeletionApproval_v1, isList: true }
   - { name: disable, type: DisableClusterAutomations_v1 }
   - { name: deleteKeys, type: string, isList: true }
+  - { name: disableKeys, type: string, isList: true }
```

Comment:
nitpick: if it is a list it should be `disabledKeys`

But I see that `deleteKeys` is already a thing so this is far from a blocker.